metadata/batch_bigquery_template_metadata.json

```diff
@@ -59,6 +59,15 @@
       "helpText": "The string name of the column that contains a datetime. The column should be of type TIMESTAMP in the SQL schema.",
       "label": "Date column"
     },
+    {
+      "name": "segment_columns",
+      "helpText": "The string names of the column that will be used for segmentation. If using more than one, it should be split by commas",
+      "label": "Segment columns",
+      "isOptional": true,
+      "regexes": [
+        "[^,]+$"
```

Comment:
I think this is what you want: `^[^,\n]*(,[^,\n]+)*$` (thx chatgpt)

The one that you have says "any sequence besides comma". But you really want "any sequence of any sequence besides comma, separated by commas".